android/src/main/java/com/onesignal/flutter/OneSignalSerializer.java

```diff
@@ -73,12 +74,25 @@ static HashMap<String, Object> convertNotificationToMap(INotification notificati
         if (notification.getAdditionalData() != null && notification.getAdditionalData().length() > 0)
             hash.put("additionalData", convertJSONObjectToHashMap(notification.getAdditionalData()));
         if (notification.getActionButtons() != null) {
-            hash.put("actionButtons", notification.getActionButtons());
+            hash.put("buttons", convertActionButtonsToMap
+            (notification.getActionButtons()));
         }
         hash.put("rawPayload", notification.getRawPayload());
         return hash;
     }
 
+    static List<HashMap<String, Object>> convertActionButtonsToMap(List<IActionButton> actionButtons) {
+        List<HashMap<String, Object>> convertedList = new ArrayList<HashMap<String, Object>>();
+        for (IActionButton actionButton : actionButtons) {
+            HashMap<String, Object> hash = new HashMap<>();
+            hash.put("i", actionButton.getId());
+            hash.put("n", actionButton.getText());
+            hash.put("icon_type", actionButton.getIcon());
```

Comment:
This should be `icon` instead of `icon_type`.

Reply:
Fixed